docs/04-operation-guides/troubleshooting/api-exposure/apix-10-api-rule-troubleshooting.md

```diff
@@ -0,0 +1,334 @@
+```
+
+This may result in an inconsistent state with missing Ory and/or Istio CRs. Your service won't be properly exposed. There are various reasons for it.
+You can check the error message of the APIRule resource, run:
+
+```bash
+kubectl get apirule -n <namespace> <api-rule-name> -o=jsonpath='{.status.APIRuleStatus}'
+```
+---
+## Issue - Missing handler configuration
+
+If your APIRule is missing required configuration, e.g. for `jwt` handler:
+
+```yaml
+spec:
+  ...
+  rules:
+    - path: /.*
+      methods: ["GET"]
+      accessStrategies:
+        - handler: jwt
+```
+
+You will get the following `APIRuleStatus` error:
+
+```
+{"code":"ERROR","desc":"Validation error: Attribute \".spec.rules[0].accessStrategies[0].config\": supplied config cannot be empty"}
+```
+
+## Remedy
+
+Supply `jwt` configuration for `trusted_issuers` or ``, e.g.:
+
+```yaml
+spec:
+  ...
+  rules:
+    - path: /.*
+      methods: ["GET"]
+      accessStrategies:
+        - handler: jwt
+          config:
+            trusted_issuers: ["https://dev.kyma.local"]
+```
+---
+## Issue - Having wrong handler configuration
+
+If your APIRule is having wrong configuration, e.g. for `jwt` handler:
+
+```bash
+cat <<EOF | kubectl apply -f -
+apiVersion: gateway.kyma-project.io/v1beta1
+kind: APIRule
+metadata:
+  ...
+spec:
+  ...
+  rules:
+    - path: /.*
+      methods: ["GET"]
+      accessStrategies:
+        - handler: jwt
+          config:
+            trusted_issuers: ["some-url"]
+EOF
+```
+
+You will get an instant error and APIRule resource won't be created:
+
+```
+The APIRule "httpbin" is invalid: spec.rules[0].accessStrategies[0].config.trusted_issuers[0]: Invalid value: "some-url": spec.rules[0].accessStrategies[0].config.trusted_issuers[0] in body should match '^(https://|file://).*$'
+```
+
+## Remedy
+
+JWT trusted issuer must be a valid `https` URL, e.g.:
+
+```yaml
+spec:
+  ...
+  rules:
+    - path: /.*
+      methods: ["GET"]
+      accessStrategies:
+        - handler: jwt
+          config:
+            trusted_issuers: ["https://dev.kyma.local"]
+```
+---
+## Issue - Unsupported handler combination
+
+If your APIRule is having unsupported handler combination on the **same** path, e.g. `allow` and `jwt` handlers:
+
+```yaml
+spec:
+  ...
+  rules:
+    - path: /.*
+      methods: ["GET"]
+      accessStrategies:
+        - handler: allow
+        - handler: jwt
+          config:
+            trusted_issuers: ["https://dev.kyma.local"]
+```
+
+You will get the following `APIRuleStatus` error:
+
+```
+{"code":"ERROR","desc":"Validation error: Attribute \".spec.rules[0].accessStrategies.accessStrategies[0].handler\": allow access strategy is not allowed in combination with other access strategies"}
+```
+
+## Remedy
+
+You should decide to `allow` access to the specific path or restrict it via `jwt` security token. Both at the same time is not allowed.
+
+---
+## Issue - Using Istio `jwt` handler configuration for Ory `jwt` handler
```

Comment:
I assume that we shouldn't mention Istio JWT (yet). Maybe we should update this troubleshooting guide with configuration errors for handlers when we will announce the feature for customers?

Reply:
This is a good point. @strekm  what do you think? Should we wait for Istio `jwt` feature or first make this troubleshooting guide for non-Istio `jwt` cases and extend afterwards?

Reply:
@videlov yes, please remove this point for now